### PR TITLE
chore: migrate to cli-table3

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

**Comment by satazor** [comment]
@DanielRuf could you check why the CI is failing? Also you might want to rebase with master.

**Comment by DanielRuf** [comment]
I'm on it.

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 language: node_js
 node_js:
   - node
-  - lts/*
+  - 10
```
Comment (satazor) [inline .travis.yml]: Why was this necessary?

**Comment by satazor** [comment]
@DanielRuf does this solves https://github.com/npms-io/npms-cli/issues/18?

File: .travis.yml (hunk 1 above)
Comment (DanielRuf) [reply .travis.yml]: Because `lts/*` just picks the last LTS branch but there are Node.js 6, 8 and 10 as LTS branch so 6 and 8 are not tested with `lts/*`.

Comment (DanielRuf) [reply .travis.yml]: Regarding this (6 is still supported until April 2019), should we also test on Node.js 6?

**Comment by DanielRuf** [comment]
Not sure what exactly you mean because cli-table3 is compatible with cli-table2 and personally I do not know if this was proposed as PR and merged in cli-table3 (if there is any patch).

The upgrade to cli-table3 mainly solves vulnerabilities and is actively maintained.

**Comment by satazor** [comment]
@DanielRuf alright, was just checking :p

File: .travis.yml (hunk 1 above)
Comment (satazor) [reply .travis.yml]: I would leave it like this.

**Comment by satazor** [comment]
Thanks!
